src/main/kotlin/lotto/model/LottoResult.kt

```diff
@@ -0,0 +1,30 @@
+package lotto.model
+
+const val MINIMUM_MATCHES = 3
+
+class LottoResult(private val winningNumbers: List<Int>, private val tickets: List<TicketModel>) {
+    fun calculateResult(): Map<Int, Int> {
+        return tickets.groupingBy { ticket ->
+            ticket.numbers.count { it in winningNumbers }
+        }
+            .eachCount()
+            .filterKeys { it >= MINIMUM_MATCHES }
+    }
```

Comment:
If we work on this Lotto Application with other developers, will they be able to understand what `Map<Int, Int>` means?